src/parser/warlock/destruction/modules/talents/Flashover.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatThousands } from 'common/format';
+
+import StatisticListBoxItem from 'interface/others/StatisticListBoxItem';
+
+const DAMAGE_BONUS = 0.25;
+const MAX_STACKS = 4;
+const STACKS_PER_CAST = 2;
+const debug = false;
+
+class Flashover extends Analyzer {
+  _currentStacks = 0;
+  bonusStacks = 0;
+  wastedStacks = 0;
+  damage = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.FLASHOVER_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (event.ability.guid !== SPELLS.CONFLAGRATE.id) {
+      return;
+    }
+    this.damage += calculateEffectiveDamage(event, DAMAGE_BONUS);
+  }
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.CONFLAGRATE.id) {
+      return;
+    }
+    if (this._currentStacks <= MAX_STACKS - STACKS_PER_CAST) {
+      // We don't waste the base Backdraft stack, nor the bonus one
+      this._currentStacks += STACKS_PER_CAST;
+      this.bonusStacks += 1;
+    }
+    else {
+      // Conflagrate grants 2 stacks with Flashover, we can attribute 1 to the base Conflagrate and 1 to the Flashover, in this case the Flashover one would get wasted
+      debug && this.log('WASTE');
+      this._currentStacks = MAX_STACKS;
+      this.wastedStacks += 1;
+    }
```

Comment:
Is it possible to waste both stacks?

Reply:
It is, when you Conflagrate on 4 stacks , but I only count the bonus one from the talent (the other wasted stack is "base"). Counting of all wasted stacks is done in Backdraft module.